features/bootstrap/Contexts/Integration/PaymentOrderContext.php

```diff
@@ -35,18 +33,22 @@ class PaymentOrderContext implements Context
     /**
      * @var null|string
      */
-    private $merchantRef= null;
-    /**
-     * @var Auth
-     */
-    private $result;
+    private $merchantRef = null;
     /**
      * @var string
      */
     private $clearpayErrorCode;
     /** @var Refund */
     private $resultRefund;
+    /** @var Payment */
+    private $resultPayment;
+    /**
+     * @var string
+     */
     private $orderId;
+    /**
+     * @var string
+     */
```

Comment:
Small CS thing here but should these either all be one line or all multiline, seems odd having a mix